src/setup-beam.js

```diff
@@ -192,13 +192,16 @@ function requestedVersionFor(tool, version, originListing, mirrors) {
   )
 }
 
-const knownBranches = ['main', 'master', 'maint']
+const knownBranches = ['master', 'maint', 'main']
```

Comment:
Changed `main` to the end because it's part of `maint` and I wanted to do the replacement in a quick fashion, not overcomplicating the algo.